contracts/AcceleratingDistributor.sol

```diff
@@ -475,4 +445,42 @@ contract AcceleratingDistributor is ReentrancyGuard, Ownable, Multicall {
             stakingTokens[stakedToken].cumulativeStaked
         );
     }
+
+    function _unstake(address stakedToken, uint256 amount) internal {
```

Comment:
Do we care about docstrings for these internal functions? We have them in place on the others, but there's overlap in the functionality with the external wrappers for these.

Reply:
We usually do not add them, only comments usually with `//`